lib/rack/recursive.rb

```diff
@@ -14,11 +14,11 @@ def initialize(url, env={})
       @url = URI(url)
       @env = env
 
-      @env[PATH_INFO] =       @url.path
-      @env[QUERY_STRING] =    @url.query  if @url.query
-      @env[HTTP_HOST] =       @url.host   if @url.host
+      @env['PATH_INFO'] =       @url.path
+      @env['QUERY_STRING'] =    @url.query  if @url.query
+      @env['HTTP_HOST'] =       @url.host   if @url.host
       @env["HTTP_PORT"] =     @url.port   if @url.port
```

Comment:
Do you know if there is a difference between  single and double quotes?

Maybe change this one too, just to stay consistent.